includes/emails/class-wc-email.php

```diff
@@ -289,7 +289,12 @@ public function get_recipient() {
 	 * @return string
 	 */
 	public function get_headers() {
-		return apply_filters( 'woocommerce_email_headers', "Content-Type: " . $this->get_content_type() . "\r\n", $this->id, $this->object );
+		if ( $this->id == 'new_order' ) { 
```

Comment:
@claudiosmweb,

I have applied changes for the same.

Below are my pull request:
https://github.com/woothemes/woocommerce/pull/11723

Commit URL:
https://github.com/woothemes/woocommerce/pull/11723/commits/7505f16893100ada31e20d12593d424f0bb3c790

Thanks,